app/assets/javascripts/oxalis/model/reducers/skeletontracing_reducer.js

```diff
@@ -82,29 +80,31 @@ function SkeletonTracingReducer(state: OxalisState, action: ActionType): OxalisS
           });
         });
       const activeTreeId = Utils.toNullable(activeTreeIdMaybe);
+      const annotationInfo = convertServerAnnotationToFrontendAnnotation(action.annotation);
 
       const skeletonTracing: SkeletonTracingType = {
-        annotationId: action.annotation.id,
         createdTimestamp: action.tracing.createdTimestamp,
         type: "skeleton",
         activeNodeId,
         cachedMaxNodeId,
         activeTreeId,
-        restrictions,
         trees,
         treeGroups: action.tracing.treeGroups || [],
-        name: action.annotation.name,
-        tracingType: action.annotation.typ,
-        tracingId: action.annotation.content.id,
+        tracingId: action.tracing.id,
         version: action.tracing.version,
         boundingBox: convertServerBoundingBoxToFrontend(action.tracing.boundingBox),
         userBoundingBox: convertServerBoundingBoxToFrontend(action.tracing.userBoundingBox),
-        isPublic: action.annotation.isPublic,
-        tags: action.annotation.tags,
-        description: action.annotation.description,
       };
 
-      return update(state, { tracing: { $set: skeletonTracing } });
+      const oldTracing = state.tracing;
+      const newTracing = {
+        // todo: do this spread more elegantly?
+        ...oldTracing,
+        ...annotationInfo,
```

Comment:
So the properties of `oldTracing` are overwritten by the properties of `annotationInfo` in this spread? So this is basically `Object.assign({}, oldTracing, annotationInfo, { skeleton: skeletonTracing })`? :)